quiche/src/h3/mod.rs

```diff
@@ -2127,6 +2137,64 @@ fn grease_value() -> u64 {
     31 * n + 33
 }
 
+/// Try to parse Extensible Priority field value.
+///
+/// The field value is expected to be a Structured Fields Dictionary; see
+/// https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-priority-12.
+///
+/// If the `u` or `i` fields are contained with correct types, a constructed
+/// Priority object is returned. Note that urgency values outside of valid range
+/// (0 through 7) are clamped to 7.
+///
+/// If the `u` or `i` fields are contained with the wrong types,
+/// Error::Done is returned.
+///
+/// Omitted parameters will yield default values.
+#[cfg(feature = "sfv")]
+pub fn parse_extensible_priority(priority: &[u8]) -> Result<Priority> {
```

Comment:
This could probably be implemented as a [`TryFrom<&[u8]>`](https://doc.rust-lang.org/std/convert/trait.TryFrom.html) trait on the `Priority` type instead of a function on its own.